main.py

```diff
@@ -0,0 +1,31 @@
+from headToHead import getHeadToHeadWinner
+from teamRankings import getTeamRankingWinner
+from toss import generateList, tossEvalTeams
+
+
+def predictMatchResult():
```

Comment:
We need to add venue now that Ayush's code is complete

Reply:
You will need to get code from Ronak's World cup branch

He modifies this function to accept 4 parameters : team1, team2, tossResult, venue

We will need to use these 4 input parameters. Only  if these 4 are not given, should we take input from console